live-cache.js

```diff
@@ -104,7 +121,7 @@
 
 	function liveCacheCallbackGenerator(lc) {
 		return function (k, v) {
-			$(lc.auto_updates[k]).html(v);
+			$(lc.auto_updates[k]).text(v);
```

Comment:
For anybody looking at this and wondering about the reasoning, using `.html()` is a potential security issue in cases in which we're receiving remote data and putting it into the DOM. We're going to use `.text()` to avoid accidentally injecting undesirable HTML. If you *do* need to inject HTML, you can still override the callback after loading livecache:

```js
window.Live_Cache.setCallback( 'key', function( key, value ) {
  // Ideally, you'll be more deliberate about creating DOM elements specifically and adding text to them, but this is just a demonstration
  jQuery( key ).html( value );
} );
```